frontend/public/scripts/users.js

```diff
@@ -0,0 +1,74 @@
+import { getContent } from "./getContent.js";
+const apiUrl = "http://localhost:4000/api/v1/users";
+getContent(apiUrl);
+
+document.addEventListener("DOMContentLoaded", () => {
+  const msg = document.querySelector(".msg");
+  const createUserForm = document.getElementById("create-attendant");
+  const getCookieValue = (cookieName) => {
```

Comment:
I see this getCookieValue being re-written in many places. In programming, you have something called DRY(do not repeat yourself). You need to put the function in a different file and import to use wherever you need it.